libbpf-tools/biolatency.bpf.c

```diff
@@ -53,9 +57,15 @@ int trace_rq_start(struct request *rq, int issue)
 	u64 ts = bpf_ktime_get_ns();
 
 	if (filter_dev) {
-		struct gendisk *disk = BPF_CORE_READ(rq, rq_disk);
+		struct request_queue *queue = BPF_CORE_READ(rq, q);
+		struct gendisk *disk;
 		u32 dev;
 
+		if (LINUX_KERNEL_VERSION >= KERNEL_VERSION(5, 17, 0))
```

Comment:
We can fix this without kconfig version check. Please refer to #3691 .

Reply:
done, thanks.